blankwindow.py

```diff
@@ -2,15 +2,33 @@
 pygame.init()
 window_width,window_height=1280,720
 display_surface=pygame.display.set_mode((window_width,window_height))
+
 #setting the screen title
 pygame.display.set_caption('Hit-Get')
 
+#importing images
+ship_surf = pygame.image.load('C:/Fall 2022/SE/hit-get/graphics/ship.png').convert_alpha()
```

Comment:
Good to see some kind of player. But any reason to choose a ship in specific?

Reply:
As this image has pointed edge I felt it looks cool. Also, as I have designed the  game window background as space, I felt space ship would look cool.